javaslang/src/test/java/javaslang/collection/HashMapTest.java

```diff
@@ -22,18 +22,18 @@ protected String className() {
     }
 
     @Override
-    <T1, T2> java.util.Map<T1, T2> javaEmptyMap() {
+    <T1, T2> java.util.HashMap<T1, T2> javaEmptyMap() {
```

Comment:
\[Checkstyle\] ERROR: Declaring variables, return values or parameters of type 'java\.util\.HashMap' is not allowed\.